install.txt

```diff
@@ -4,12 +4,12 @@ The directory corresponding to your cart's version will be called
 YOUR_VERSION.  For example, if you are running Zen Cart 1.5.5, 
 YOUR_VERSION is 1_5_5. 
 
-1) In YOUR_VERSION, change the directory names YOUR_TEMPLATE and 
-YOUR_CLASSIC_RESPONSIVE to correspond to what's being used in your cart.
+1) In includes, change the directory name YOUR_TEMPLATE 
+to correspond to what's being used in your cart.
 Then copy all the files in this directory to your cart.
 
-2) In includes, change the directory name YOUR_TEMPLATE 
-to correspond to what's being used in your cart.
+2) In YOUR_VERSION, change the directory names YOUR_TEMPLATE and 
+YOUR_CLASSIC_RESPONSIVE to correspond to what's being used in your cart.
```

Comment:
This (YOUR_CLASSIC_RESPONSIVE) was added in the event someone "copies" the classic_responsive directory to create their own template (in ZC 1.5.5+ only). Not sure I have a good suggestion of "words to use" but one could read this and try to change both directories to the same folder name. :(